seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/Firefox.scala

```diff
@@ -11,6 +11,12 @@ import scala.collection.JavaConversions._
 import scala.collection.mutable.ArrayBuffer
 
 object Firefox extends SeleniumBrowser {
+
+  private final val popHijackedConsoleScript = {
+    "var callback = arguments[arguments.length - 1];" +
```

Comment:
It does not end up in global scope, it is inserted in a closure that those `(function() { ... }).call(this);` and some other stuff to set up its own small environment.